src/containers/layouts/ecommerce/shoppingCart/shoppingCart.container.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { NavigationScreenProps } from 'react-navigation';
+import { Product } from '@src/core/model';
+import { ShoppingCart } from './shoppingCart.component';
+import { products } from '@src/core/data/product';
+import { CartProduct } from '@src/containers/layouts/ecommerce/shoppingCart/cartProduct.model';
+
+interface State {
+  cartProducts: CartProduct[];
+}
+
+export class ShoppingCartContainer extends React.Component<NavigationScreenProps, State> {
+
+  private prepareProducts = (): CartProduct[] => {
+    return [products[0], products[1]]
+      .map((item: Product) => ({
+        ...item,
```

Comment:
much better if such cases are formatted either mapping function or has `return` statement, which is omitted here

Reply:
Done